bin/cli.js

```diff
@@ -4,10 +4,14 @@
 
```

Comment:
Is there a reason why you added this logic here instead of `bootstrap.js`?

Reply:
yep, bin/cli.js - is the entry point which includes logic of preparing data for webpack-cli and executes webpack-cli
boostrap - is the entry point of webpack-cli

Reply:
This change make sense to me,
`bootstrap.js` would contain logic to running CLI and this file will run CLI as a child process.

Reply:
why bin/cli.js can't exec boostrap.js?

Reply:
i think that bin/cli.js is a good place for it

Reply:
Is there any problem? I too think `bin/cli.js` is a good place.

Reply:
Just a lost in translation misunderstanding :)
Thank you for the explanation!